tox.ini

```diff
@@ -44,13 +43,13 @@ extras =
 deps =
     xdist: pytest-xdist
     oldestdeps: minimum_dependencies
-commands_pre =
+commands_pre = 
     oldestdeps: minimum_dependencies jwst --filename {toxinidir}/requirements-min.txt
     oldestdeps: pip install -r {toxinidir}/requirements-min.txt
-    stdevdeps: pip install -r {toxinidir}/requirements-dev-st.txt -U --upgrade-strategy eager
-    devdeps: pip install -r {toxinidir}/requirements-dev-thirdparty.txt -U --upgrade-strategy eager
-    pip freeze
+    devdeps: pip install --ignore-installed -r {toxinidir}/requirements-dev.txt -U --upgrade-strategy eager
 commands =
+    devdeps: pip install numpy>=0.0.dev0 -U --extra-index-url https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
+    pip list
```

Comment:
Why `list` and not `freeze` here?

Reply:
`list` gives the same information as `freeze` but in a nicer format for human reading (we're not parsing the logs)

Reply:
But pip freeze let you copy paste to requirements file though, if we never need it (but I never did...).